packages/TelegramClientTests-Core.package/TCTCCoreTests.class/instance/testLoggedInUserID.st

```diff
@@ -0,0 +1,9 @@
+testing
+testLoggedInUserID
+
+	self mockTeleClient onRequestType: 'getMe' respond: TCTMocks mockUserJson.
+	self mockTeleClient logIn.
+	
+	1 seconds wait.
+	
+	self assert: TCTMocks mockUser id equals: (self core loggedInUserID).
```

Comment:
We compare the id of the mockUser with the id from the mockUserJson right? 
So the id has to be and stay the same in 2 places. Maybe it would be better to compare the id of the logged in user to the Json at 'id'

Reply:
For creating the `mockUserJson`, all the data of the `mockUser` is used. Thus there is only one place for the `mockUser id`.
```
mockUserJson
	^ JsonObject newFrom: {
		'@type' -> 'user'.
		'id' -> self mockUser id.
		'first_name' -> self mockUser firstName.
		'last_name' -> self mockUser lastName.
		'username' -> self mockUser username.
		'@extra' -> self mockUser id asString
	}
```

So I guess that is no problem anymore?

Reply:
yeah, I hadn't noticed that